config/donation-settings.js

```diff
@@ -8,7 +8,8 @@ const amountsConfig = {
   onetime: [2500, 5000, 7500, 10000, 15000]
 };
 const defaultAmount = {
-  month: 500
+  month: 500,
+  onetime: 500
```

Comment:
Ahmad: To confirm...

Reply:
We are currently offering 5 dollar monthly and that should be the only default amount.